onnxruntime/server/environment.h

```diff
@@ -20,11 +21,12 @@ class ServerEnvironment {
 
   OrtLoggingLevel GetLogSeverity() const;
 
-  const Ort::Session& GetSession() const;
-  void InitializeModel(const std::string& model_path);
-  const std::vector<std::string>& GetModelOutputNames() const;
+  const Ort::Session& GetSession(const std::string& model_name, const std::string& model_version) const;
+  void InitializeModel(const std::string& model_path, const std::string& model_name, const std::string& model_version);
```

Comment:
Can `model_name` different from the model file name in the `model_path`?

Reply:
Should we restrict the `model_version` be integer here?

Reply:
For now we're only supporting the default/1 - my thought process was that eventually we'd support some kind of repository. My choice was `string, string` because that's what the executor takes (I think stemming from the regex).